README.md

```diff
@@ -4,7 +4,9 @@ This is a plugin for a local DynamoDB instance using Docker.
 
 When you are using Architect's sandbox mode, the plugin starts a Docker container running the [`amazon/dynamodb-local:latest`](https://hub.docker.com/r/amazon/dynamodb-local) image.
 
-This can be used alongside other plugins such as [@hicksy/arc-plugin-sandbox-stream](https://github.com/hicksy/arc-plugin-sandbox-stream) to allow for more realistic debugging with `tables-streams`.
+If you have defined a `@tables-streams` section in your `app.arc` file, the respective streams will be enabled.
+
+There is an issue with the Docker image of DynamoDB local that causes a `TrimmedDataAccessException` error to be thrown on the first read. This problem lies deeper within DynamoDB and is out of the scope of this plugin. To handle this issue, there is a reset function that will trigger automatically. In practice, this means that the 1st invocation will fail (you will see a logged `TrimmedDataAccessException` message in your console), but the following invocations will work successfully.
```

Comment:
```suggestion
There is an issue with the Docker image of DynamoDB local that causes a `TrimmedDataAccessException` error to be thrown on the first read. This problem lies deeper within DynamoDB and is out of the scope of this plugin. To handle this issue, there is a reset function that will trigger automatically. In practice, this means that the first invocation will fail (you will see a logged `TrimmedDataAccessException` message in your console), but the following invocations will work successfully.
```

Is there a GitHub issue or a piece of AWS documentation about this?

Reply:
Not that I have seen directly, but there are references about the problem that I have come across when it started popping up:
- https://github.com/aws/aws-sdk-java/issues/2923#issuecomment-2062361053 They claim it was addressed in a  a few releases back (2.4.0, latest is 2.6.0) but I still encounter it using the `latest` tag

- and in the repo that we originally were testing: https://github.com/hicksy/arc-plugin-sandbox-stream/issues/1